doc/source/release/notes-1.0.0.rst

```diff
@@ -73,10 +73,12 @@ Contributors
 .. _`@aglowacki`: https://github.com/aglowacki
 .. _`@dmpelt`: https://github.com/dmpelt
 .. _`@dgursoy`: https://github.com/dgursoy
+.. _`@dakefeng`: https://github.com/dakefeng
 .. _`@decarlof`: https://github.com/decarlof
 .. _`@lbluque`: https://github.com/lbluque
 .. _`@yxqd`: https://github.com/yxqd
 .. _`@michael-sutherland`: https://github.com/michael-sutherland
 .. _`@ravescovi`: https://github.com/ravescovi
 .. _`@tacaswell`: https://github.com/tacaswell
 .. _`@prjemian`: https://github.com/prjemian
+.. _`@celiafish`: https://github.com/celiafish
```

Comment:
Ah.. Please add your names to the upper part also. These are just the hyperlinks, and are not visible at website.